### DISCUSSION TITLE
Is there a way to use flavor banner with go router?

### DISCUSSION THREAD

**Discussion by ZahidTekbas** [AngeloAvv/flutter_flavorizr#239] [Q&A]
Hello all,

I am using go_router package in my application. That's why I'm using MaterialApp.router object to start my app. It doesn't let me use home object in that widget so I can't add flavor banner to my application. Is there a workaround for that?

**Comment by YukiAttano** [comment]
You can add a ShellRoute around all your routes. That way you can add a banner.
It might also be possible, that you can Wrap your MaterialApp() with a Banner.
